helevents/templates/rest_framework/event_list.html

```diff
@@ -394,6 +394,43 @@ <h4 id="">Filtering for minimum_attendee_capacity</h4>
         <pre><code>event/?minimum_attendee_capacity_lte=10</code></pre>
         <p><a href="?minimum_attendee_capacity_lte=10" title="json">See the result</a></p>
 
+        <h3 id="remaining-attendee-capacity">Remaining registration attendee or waiting list capacity</h3>
+        <p>Filters for filtering by registration remaining_attendee_capacity and remaining_waiting_list_capacity:</p>
+
+        <h4 id="">Filtering for registration remaining_attendee_capacity</h4>
```

Comment:
We could clean the empty id. I see "someone" else has also added the empty id above these changes 🙈

Reply:
Empty id attributes have been cleaned :broom: